app/backend/app.py

```diff
@@ -411,8 +416,10 @@ def create_app():
         configure_azure_monitor()
         # This tracks HTTP requests made by aiohttp:
         AioHttpClientInstrumentor().instrument()
-        # This tracks HTTP requests made by httpx/openai:
+        # This tracks HTTP requests made by httpx:
         HTTPXClientInstrumentor().instrument()
```

Comment:
Is this one still tracing anything? I haven't seen traces from it recently, likely due to the OpenAI regression, and it looks like httpx is only required by openai in this codebase. I'm wondering if it can be removed.

Reply:
Ping on this question.

Reply:
Anthony says that httpx works sometimes, and maybe will work consistently in future, and its fine to have multiple spans, so just keep it in.